x11/xcowsay/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nickg xcowsay 1.4 v
+revision            1
+github.tarball_from releases
+name                xcowsay
+categories          x11
+license             GPL-3+
+platforms           darwin
+maintainers         nomaintainer
+description         X11 xcowsay app
+long_description    xcowsay displays a cute cow and message on your desktop. \
+                    It was inspired by the original cowsay.
+
+checksums           rmd160  d9e7cdc0c52f6ae03aa9eb7b967bc79c768c2822 \
+                    sha256  c7e261ba0262c3821c106ccb6d6f984e3c2da999ad10151364e55d1c699f8e51
+
+depends_build       port:pkgconfig
```

Comment:
Use `depends_build-append` instead to not overwrite the GitHub portgroup dependencies.